pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix

```diff
@@ -14,33 +14,19 @@
 }:
 
 let
-  libdeltachat' = libdeltachat.overrideAttrs (old: rec {
```

Comment:
Do we now use the binary version provided by deltachat?

Reply:
That version would only work on x86_64-linux and x86_64-darwin.